packages/atomic/src/components/atomic-smart-snippet/atomic-smart-snippet-answer.tsx

```diff
@@ -15,14 +15,38 @@ export class AtomicSmartSnippetAnswer {
   @Prop({reflect: true}) htmlContent!: string;
   @Prop({reflect: true}) innerStyle?: string;
 
-  @Event({
-    eventName: 'atomic/smartSnippet/answerRendered',
-  })
-  private answerRendered!: EventEmitter<{height: number}>;
+  @Event({bubbles: false})
+  private answerSizeUpdated!: EventEmitter<{height: number}>;
   private wrapperElement?: HTMLElement;
+  private isRendering = true;
+  private resizeObserver: ResizeObserver | undefined;
```

Comment:
I decided to use a `ResizeObserver` instead of the `window.onresize` event or the `ScreenOrientation.onchange` event since technically the size of the smart snippet could also change if a parent element is resized, which I imagine could easily be the case in some clients' cases.